SwiftWisdom/Core/UIKit/NibLoadableView.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  NibLoadableView.swift
+//  SwiftWisdom
+//
+//  Created by Christian Howland on 3/31/17.
+//  Copyright © 2017 Intrepid. All rights reserved.
+//
+
+import UIKit
+
+/**
+ Subclassing NibLoadableView allows you to automate the loading of a Nib into a placeholder view inside
+ another Nib, using only the Custom Class attribute.
+ 
+ It removes the need to perform the following manual steps every time you want a Nib that can load into a placeholder View:
+
+ - Setting an outlet for the top level view
+ - Setting 'translatesAutoresizingMaskIntoConstraints' to false
+ - Adding constraints to fit the new view into its container
+ - Writing boilerplate Nib loading code.
+
+ The minimum amount of code required to use this would just be the inheritance line, i.e.:
+
+ " YourCustomView: NibLoadableView {} "
+
+*/
+
+class NibLoadableView : UIView {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Colon Violation: Colons should be next to the identifier when specifying a type and next to the key in dictionary literals. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Acolon)